lib/preferences.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:shared_preferences/shared_preferences.dart';
+import 'dart:convert';
+
+
+Map<String, bool> _hirasol = {
+  'あ い う え お': true,
+  'か き く け こ': true,
+  'さ し	す せ そ': true,
+  'た ち つ て と': true,
+  'な に ぬ ね の': true,
+  'は ひ ふ へ ほ': true,
+  'ま み む め も': true,
+  'や ゆ よ': true,
+  'ら り る れ ろ': true,
+  'わ を': true,
+  'ん': true
+};
+
+Map<String, bool> _katasol = {
+  'ア イ ウ エ オ': true,
+  'カ キ ク ケ コ': true,
+  'サ シ ス セ ソ': true,
+  'タ チ ツ テ ト': true,
+  'ナ ニ ヌ ネ ノ': true,
+  'ハ ヒ フ ヘ ホ': true,
+  'マ ミ ム メ モ': true,
+  'ヤ ユ ヨ': true,
+  'ラ リ ル レ ロ': true,
+  'ワ ヲ': true,
+  'ン': true,
+};
+
+  final String _hirasolSet =  "hiragana";
+  final String _katasolSet = "katakana";
+
+  /// ------------------------------------------------------------
+  /// Method that returns the Hiragana set
+  /// ------------------------------------------------------------
+  Future<String> getHiraganaSet() async {
+    final SharedPreferences prefs = await SharedPreferences.getInstance();
```

Comment:
I think it would be better to do this only once, store the `SharedPreferences` instance as a global

Reply:
Changed!

Reply:
Changed!